kerchunk/intro.qmd

```diff
@@ -0,0 +1,40 @@
+---
+title: "Kerchunk"
+subtitle: References for Chunked (maybe), Compressed (maybe) N-Dimensional Arrays
+---
+
+## What is Kerchunk?
+
+Kerchunk's potential utility to the geospatial data community cannot be overstated. Kerchunk enables cloud-optimized access to traditional geospatial file formats, like NetCDF, potentially negating the need to create and store copies of data for cloud-optimized access.
+
+Kerchunk creates Zarr metadata, stored in a "reference file", for non-Zarr data. Kerchunk enables a unified way to access chunked, compressed n-dimensionsional data across a variety of conventional data formats. The Kerchunk library now supports NetCDF/HDF5, GRIB2, TIFF.
+
+Learn more about Kerchunk at [kerchunk.readthedocs.io](https://kerchunk.readthedocs.io/en/latest/).
+
+## How to Kerchunk 
```

Comment:
Where's the How part? I just see an example reference file, nothing about use the `Kerchunk` library/tools to scan an archival format to generate the reference file.